common/src/main/java/org/corfudb/common/protocol/client/ChannelHandler.java

```diff
@@ -223,12 +228,14 @@ private void checkRequestTimeout() {
 
     protected <T> CompletableFuture<T> sendRequest(Request request) {
         requestLock.readLock().lock();
+
         try {
             checkArgument(request.hasHeader());
             Header header = request.getHeader();
             CompletableFuture<T> retVal = new CompletableFuture<>();
             pendingRequests.put(header.getRequestId(), retVal);
             ByteBuf outBuf = PooledByteBufAllocator.DEFAULT.buffer();
+            outBuf.writeByte(0x2); // Temporary -- Add Corfu msg marker indicating new message type
```

Comment:
I think 0x2 and 0x1 has been converted to statics. Rebase on @zfrenette  latest branch

Reply:
Yes, Will do.